avalonia/ReactiveUI.Samples.Suspension/ReactiveUI.Samples.Suspension.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Update="**\*.xaml.cs">
+      <DependentUpon>%(Filename)</DependentUpon>
+    </Compile>
+    <AvaloniaResource Include="**\*.xaml">
+      <SubType>Designer</SubType>
+    </AvaloniaResource>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Avalonia" Version="0.8.1-cibuild0003100-beta" />
```

Comment:
Version ranges so dependabot doesn't explode every week?

Reply:
Avalonia API is extremely unstable now, so if I set `Version` to e.g. `*` there will be a chance that the project won't compile someday x) Btw just curious, is dependabot able to resolve MyGet CI feeds from `nuget.config` file format?